control/veda_heal.py

```diff
@@ -106,18 +105,20 @@ def send_encodes(self):
                 veda_id = v.edx_id
                 encode_profile = encode
                 job_id = uuid.uuid1().hex[0:10]
-                task_result = celeryapp.worker_task_fire.apply_async(
-                    (veda_id, encode_profile, job_id),
-                    queue=self.auth_dict['celery_worker_medium_queue'].strip(),
-                    connect_timeout=3
-                )
+                # TODO: Commenting this out temporarily.  The worker_task_fire task
```

Comment:
For the official record, why did this need to be commented out / what was breaking with it included?

Reply:
We had a circular dependency between veda_heal.py and celeryapp.py (both were importing each other), and this was causing errors when a 3rd module tried to import veda_heal.  To remove the dependency on celeryapp I removed the call above.